src/index.ts

```diff
@@ -91,6 +94,15 @@ export function viteSvgToWebfont<T extends GeneratedFontTypes = GeneratedFontTyp
             }
             return resolvedVirtualModuleId;
         },
+        generateBundle(_, bundle) {
+            for (const { type, href } of tmpGeneratedWebfonts) {
+                for (const chunk of Object.values(bundle)) {
+                    if (chunk.name && href.endsWith(chunk.name)) {
```

Comment:
Unfortunately, I can't find a way to get bundled asset filename by its entry name so here's the workaround where we're comparing the generated asset names with the fonts we generated earlier.